pytorch_to_returnn/torch/__init__.py

```diff
@@ -4,6 +4,7 @@
 from .nn.functional import *
 from .serialization import load
 from .autograd import *
-from .jit import *
-from .onnx import *
 from . import nn
+import torch.cuda
+import torch.onnx
+import torch.jit
```

Comment:
Why do you do this?
Is this done in PyTorch?

Reply:
Also, this is wrong in any case.

Reply:
This is done in PyTorch, see https://github.com/pytorch/pytorch/blob/master/torch/__init__.py#L570
That's why I took it over.